packages/ui/src/primitives/tailwind/Modal/index.tsx

```diff
@@ -112,6 +121,7 @@ export const ModalFooter = ({
 }
 
 const Modal = ({
+  id,
```

Comment:
this may not be an ideal case
footer can be hidden using the `hideFooter` prop
and also `id` on modal should go inside the modal's direct children

Reply:
I'm not entirely sure if I understood you correctly, but are you suggesting that I hide the footer and create a custom one inside the modal content? Honestly, I don't think it's really necessary to do that because modals could lose their consistency in different cases, and I think it goes a bit beyond the Open/Closed Principle. However, I did change the button identifiers to data-test-id=, and I think that works. What are your thoughts?

Reply:
How about we add a `data-test-id` to [the outermost div](https://github.com/ir-engine/ir-engine/blob/87ecd255f0c6a2420ac9a911c07183fa4f250fd6/packages/ui/src/primitives/tailwind/Modal/index.tsx#L131) ?

This div is always present and can be used for testing.